css/layout-helpers.css

```diff
@@ -5,3 +5,12 @@
   align-items: center;
   justify-content: center;
 }
+/*Use to caption images in EdX*/
+.figcaption {
+  font-family: Helvetica, Arial, sans-serif;
```

Comment:
Seeing as `.figcaption` is a class, I assume it will be used on a `<span>` or a `<p>`? Let's add that to your comment above (indication of what HTML element you should use this on). Also, because it will be inheriting styles from `<span>` or `<p>`. we should maybe drop the `font-family` rule because a) I think it's wrong (shouldn't it be Helvetica Neue?) and b) because it should come through from the base element

Reply:
I think you talked about this already but just a reminder `<figure>` and `<figcaption>` are elements, not classes :)

Reply:
Amended ".figcaption" to just "figcaption" after some research, cannot find examples of <p> or <span> used with figcaps.
Corrected the font-family to "Helvetica Neue". As figcap is an element, not sure it will inherit from the base element?

Reply:
Also
@meelijane, you mentioned that "display:table" wasn't quite right for the "figure" style. Chat when you have a sec? Will research and try to fix myself in the meantime.